scripting/cwx/loadout_radio_menu.sp

```diff
@@ -354,6 +357,15 @@ static int OnEquipMenuEvent(Menu menu, MenuAction action, int param1, int param2
 			if (!uid[0]) {
 				FormatEx(itemName, sizeof(itemName), "%t", "UnequipCustomItem");
 				redraw = true;
+			} else if(item.localizedNames) {
+				char lang[16], text[MAX_ITEM_IDENTIFIER_LENGTH];
+				GetLanguageInfo(GetClientLanguage(client), lang, sizeof(lang), text, sizeof(text));
```

Comment:
Just making sure, `text` is supposed to temporarily hold the buffer for the new name, right?  Will change this to `MAX_ITEM_NAME_LENGTH`.  No point in using it with `GetLanguageInfo` since the buffers for full language name are optional.